scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html

```diff
@@ -285,7 +285,9 @@ <h5>
   $(document).ready(function(){
     setChromosomes();
     populateSubTypeSelect('add_variant');
-    populateSubTypeSelect('modify_variant')
+    {% for variant in managed_variants %}
+      populateSubTypeSelect('modify_variant_'.concat('{{variant.variant_id|safe}}'));
```

Comment:
Exactly! Thanks!

Just out of curiosity, why the `concat()` and not just placing the jinja lookup in the string?

Reply:
No particular reason. And actually it's simpler without concat, I'll switch to your option!